source/sass/block/programs.scss

```
@@ -0,0 +1,71 @@
+.programs {
+  display: flex;
+  flex-direction: column;
+}
+
+.program__item {
+  background-color: var(--basic_grey);
+  padding: 21px 33px 18px 22px;
+}
+
+.program__title {
+  position: relative;
+  margin: 0;
+  padding: 30px 0 0 0;
+  margin-bottom: 22px;
+  font-size: 24px;
+  line-height: 37px;
+  font-weight: 400;
+  text-transform: uppercase;
+  color: var(--title_black);
+  text-align: center;
+}
+
+.program__group {
+  display: flex;
+  align-items: center;
+}
+
+.program__circle {
+  width: 45px;
+  height: 45px;
+  background-color: var(--basic_green);
+  border-radius: 50%;
+}
+
+.program--slim-title::before {
```

Модификаторы пишем через амперсанд при этом эти правила надо перенести выше в блок програм, если его нет- то создать
```suggestion
&--slim-title {
  &::before {
```

---

исправлено